docs/metrics.md

```diff
@@ -8,7 +8,7 @@ $ helm repo add prometheus-community https://prometheus-community.github.io/helm
 $ helm repo update
 $ helm install prometheus prometheus-community/kube-prometheus-stack
 ```
-2. Enable metrics by setting `enableMetrics: true` in [values.yaml](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/values.yaml).
+2. Enable metrics by configuring `controller.enableMetrics` or `node.enableMetrics`.
```

Comment:
Consider replacing 'or' with 'and' to make it clearer that you need to separately enable both.

Or add a sentence about how controller  metrics measure x, and node metrics measure y.

Reply:
Fixed ✅